setup.py

```diff
@@ -30,12 +30,15 @@ def read(fname):
                         'seaborn>=0.10.0',
                         'cloudpickle>=0.7.0',
                         'h5py>=2.10.0',
+                        'amici @ git+https://github.com/amici-dev/amici'
```

Comment:
Do we really want to hardcode the dependency on the AMICI develop branch in our current branch? @yannikschaelte Any opinions? :)

Reply:
I would suggest to first merge amici.

Reply:
Yep I agree :+1: Meant that in my earlier comment

> merging will be delayed as this PR requires the next AMICI release

`develop` is just used now to check that everything will work. Similarly, the current edit to ".travis.yml" will also be removed from this PR before merging.